lib/filegen/template.go

```diff
@@ -44,6 +45,30 @@ func (m *Manager) registerTemplateFileForPath(pathname string,
 	return nil
 }
 
+func createFuncMap() template.FuncMap {
+	return template.FuncMap{
+		"GetSplitPart": func(s string, sep string, index int) string {
+			parts := strings.Split(s, sep)
+			if index >= 0 && index < len(parts) {
+				return parts[index]
+			}
+			return ""
+		},
+		"ToLower": func(s string) string {
```

Comment:
There's no need to wrap these library functions; they can be used directly as they have the same behaviour.